docs/en/_sidebar.md

```diff
@@ -857,8 +857,8 @@
   - [Licenses](/contribute/licenses.md)
 - [Releases](/releases/index.md)
   - [main (alpha)](/releases/main.md)
-  - [1.16 (release candidate)](/releases/1.16.md)
```

Comment:
FYI this file never needs to change (it is "for future work" and I update it myself automatically at various points).